var/spack/repos/builtin/packages/nfft/package.py

```diff
@@ -25,7 +25,7 @@
 from spack import *
 
 
-class Nfft(AutotoolsPackage):
+class Nfft(Package):
```

Comment:
Because we are overriding `install`, see @davydden's comment, and for consistency with the fftw package.